### PR TITLE
Mike

### PR DESCRIPTION
**Pull request by tsjuder1**
All done, please review

### CODE REVIEW

**Review by AnnaLoshak — APPROVED** [review]
Good work

File: src/test/java/Group4/MikeProject1.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    for (int j = 0; j < words.length; j++)
+                        if (word2.equals(words[j]))
+                            if (Math.abs(j - i) < distance)
+                                distance = Math.abs(j - 1);
+
+
+        return distance;
+    }
+
+}
```
Comment (BadamkhandB) [inline src/test/java/Group4/MikeProject1.java:26]: nice done

**Review by BadamkhandB — APPROVED** [review]

**Review by ZaierAouani — APPROVED** [review]
Great!

**Review by msebbata — APPROVED** [review]
looks good

**Review by KarinaFrei — APPROVED** [review]
Great!

**Comment by ZaierAouani** [comment]
Good.

**Review by VeronikaKrasnokutska — APPROVED** [review]
Good

**Pull request closed without merging**